packages/create-react-admin/src/generateProject.ts

```diff
@@ -1,14 +1,17 @@
 import path from 'path';
 import fs from 'fs';
 import fsExtra from 'fs-extra';
-import merge from 'lodash/merge';
+import url from 'url';
+import merge from 'lodash/merge.js';
 import { ProjectConfiguration } from './ProjectState.js';
 import { generateAppFile } from './generateAppFile.js';
 import { generateAppTestFile } from './generateAppTestFile.js';
 
+const __filename = url.fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
```

Comment:
https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c#what-do-i-use-instead-of-__dirname-and-__filename